docs/architecture/modernize-desktop/whats-new-dotnet-core.md

```diff
@@ -0,0 +1,144 @@
+---
+title:  What's new with .NET Core 3 for Desktop?
+description: Learn about .NET Core, differences between .NET COre and .NET Framework and the new features tht were added in the latest version of the .NET Core - version 3.
+ms.date: 09/16/2019
+---
+
+# What's new with .NET Core 3 for Desktop?
+
+From the version 3.0, .NET Core supports Windows Forms and WPF, so now you have a choice between .NET Framework and .NET Core for your desktop applications. This chapter will describe what is .NET Core and what are it's benefits for desktop applications.
+
+## From .NET Framework to .NET Core - The motivation behind .NET Core
+
+Since its launch in 2002, .NET Framework has evolved through the years to support many technologies like Windows Forms, ASP.NET, Entity Framework, Windows Store, and many others. All of them are different in nature. Therefore, Microsoft was approaching this evolution by taking parts of the .NET Framework and creating a different application stack for each technology. That way, development capabilities could be customized for the needs of the specific stack, which maximized the potential of every platform. That lead to fragmentation on the versions of the .NET Framework maintained by different independent teams. All these stacks have a common structure, containing an App Model, a Framework, and a Runtime, but they differ in the implementation of each of these parts.
+
+If you're targeting only one of these platforms, you can use this model. However, in many cases you might need more than one target platform in the same solution. For example, your application may have a desktop admin part, a customer-facing web site that shares the back-end logic running on a server, and even a mobile client. In this case, you need a unified coding experience that can span all this .NET verticals.
+
+By the time Windows 8 was released, the concept of the Portable Class Libraries (PCL) was born. Originally, the .NET Framework was designed around the assumption that it would always be deployed as a single unit, so [factoring](http://en.wikipedia.org/wiki/Decomposition_(computer_science)) wasn't a concern. To face the problem of code sharing between verticals, the driving force was on how to refactor the framework. The idea of contracts is to provide a well-factored API surface area. Contracts are simply assemblies that you compile against and are designed with proper factoring in mind taking care of the dependencies between them.
```

Comment:
You could say Portable Class Library (PCL) here, in the singular form...

```suggestion
By the time Windows 8 was released, the concept of Portable Class Libraries (PCLs) was born. Originally, the .NET Framework was designed around the assumption that it would always be deployed as a single unit, so [factoring](http://en.wikipedia.org/wiki/Decomposition_(computer_science)) wasn't a concern. To face the problem of code sharing between verticals, the driving force was on how to refactor the framework. The idea of contracts is to provide a well-factored API surface area. Contracts are simply assemblies that you compile against and are designed with proper factoring in mind taking care of the dependencies between them.
```